content/en/docs/instrumentation/net/_index.md

```diff
@@ -18,16 +18,17 @@ Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
 
 # Status and Releases
 
-This is the current release status for OpenTelemetry components in this
-language:
+The release status for .NET OpenTelemetry components is as follows:
 
-| Tracing | Metrics | Logging |
-| ------- | ------- | ------- |
-| 1.0    | Alpha   | Beta    |
+- **Traces**: [Stable][]
+- **Metrics**: [Stable][]
+- **Logs**:
+  - [ILogger][]: [Stable][]
+  - [OTLP log exporter][]: [Experimental][]
 
 {{% latest_release "dotnet" /%}}
 
-# Further Reading
+# Learn more
```

Comment:
All: I have a question unrelated to the changes in this PR, but relevant to this page. Given that this section of the website has a [Getting started page](https://opentelemetry.io/docs/instrumentation/net/getting-started/), can we drop the **Getting Started** entry below (lines 35-36)?

If so, I can make the change in this PR, or a followup PR.

Reply:
I think so, yes

Reply:
@cijothomas @reyang - any objection to me removing the Getting Started entry?

Reply:
I can see the problem here - the [getting started page](https://opentelemetry.io/docs/instrumentation/net/getting-started/) is only talking about traces, it does not cover logs and metrics. And the doc wasn't reviewed by any .NET approver/maintainer.

Reply:
I would suggest that we keep the scope of this PR small, maybe leave it for another PR or create an issue tracking it.

Reply:
Agreed and done, see #1320.